rulesets/java/jpinpoint-rules.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ruleset name="jpinpoint-rules" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd" xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:fn="http://www.w3.org/TR/xpath-functions/">
```

Comment:
Removed is: xmlns:fn="http://www.w3.org/TR/xpath-functions/"
For what reason?

Reply:
idea says: not used, but I see it gets generated with the merge call: put the original merged file in commit

Reply:
ah, it was gone again: it is the "optimize imports" option that I have enabled in intellij: it removes unused imports at commit automatically!!

Reply:
"One more question: are all occurrences of the singleton annotations in the rules now extended with @RestController?"

I think so, I looked for all places in merged file where @Controller is used, let me know if you see an additional one.